pkg/rabbitmq/queue_runner.go

```diff
@@ -63,13 +63,18 @@ func newQueueRunner(conn *amqp.Connection, exchangeName string, q QueueSpec) (*q
 		}
 		log.Debugf("Successfully declared DLQ: %s", q.dlqName)
 		if q.dlxName != "" {
-			log.Debugf("Binding DLQ %s to DLX %s", q.dlqName, q.dlxName)
-			if err := r.ch.QueueBind(q.dlqName, "", q.dlxName, false, nil); err != nil {
+			// Use configured DL routing key if present; otherwise keep empty key
+			bindingKey := ""
+			if q.cfg.DLRoutingKey != "" {
+				bindingKey = q.cfg.DLRoutingKey
+			}
```

Comment:
```suggestion
			bindingKey := q.cfg.DLRoutingKey
```

This would have the same result.